src/hevm/src/EVM/SymExec.hs

```diff
@@ -365,9 +365,11 @@ simplify e = if (mapExpr go e == e)
     go (EVM.Types.LT (Lit a) (Lit b))
       | a < b = Lit 1
       | otherwise = Lit 0
-    go (EVM.Types.GT (Lit a) (Lit b))
-      | a > b = Lit 1
+    go (EVM.Types.GT a b) = EVM.Types.LT b a
```

Comment:
Shouldn't this rewrite to `LEq`?

Reply:
What I am trying to say I think is that `a < b` is the same as `b > a`. But actually...
```
a<b == !(!(a<b))
a<b == !(a>=b)
```

So then we could rewrite all 4 to just `LT`. All of `LT`, `GT`, `LEq`, `GEq` would become LT. Let me do that, actually. Maybe more sane? Then we'd have:

* `GT`->`LT` via symmetry
* `GEq`->`LT` via negation (as above)
* `LEq`->`GEq` via symmetry

What do you think?

Reply:
ah yes, I wasn't thinking properly. I agree that rewriting everything to LT seems nice.

Reply:
No, it was great, now it's all LT, yay :)